### DISCUSSION TITLE
httpproxy in tunnels.conf or i2pd.conf

### DISCUSSION THREAD

**Discussion by anikey-from-i2p** [PurpleI2P/i2pd#2077] [Q&A] [answered]
Is there any difference between adding `httpproxy` in i2pd.conf (as by default) and in tunnels.conf?

The latter is only once mentioned in the docs.

Is there any point in having one "blessed" http proxy (in i2pd.conf) and other ones in tunnels.conf, or are they equivalent?

I've noticed that i can't change outproxy with the one in i2pd.conf (by reloading config), it stays at the old value.

**Comment by r4sas** [comment, marked as answer]
i2pd.conf is not reloadable. Only `tunnels`, as said on menu item and docs.

Difference is that i2pd.conf httpproxy is default proxy, which can be disabled, and tunnels.conf can create new tunnels along with default one, if needed.
